### PR TITLE
added a test file

### PR DESCRIPTION
**Pull request by MiladNazeri**
bla

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+are you cat?
+am I cat?
```
Comment (RebeccaStankus) [inline test.js]: What kind of cat?

File: test.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+are you cat?
```
Comment (RebeccaStankus) [inline test.js]: I am.

File: test.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+are you cat?
+am I cat?
+are we cat?
```
Comment (RebeccaStankus) [inline test.js]: I would say so.

File: test.js (hunk 1 above)
Comment (MiladNazeri) [reply test.js]: DOES IT MATTER???

File: test.js (hunk 2 above)
Comment (MiladNazeri) [reply test.js]: Am I?

File: test.js (hunk 3 above)
Comment (MiladNazeri) [reply test.js]: cool.  checks out.

**Pull request closed without merging**